packages/labs/observers/src/intersection_controller.ts

```diff
@@ -124,8 +125,8 @@ export class IntersectionController<T = unknown> implements ReactiveController {
   }
 
   hostConnected() {
-    if (this._target) {
-      this.observe(this._target);
+    for (const target of this._targets.values()) {
```

Comment:
Sets are iterable so it's not clear why you need to use `.values()` here.

Reply:
My mistake. `values` is not needed.